.github/workflows/ci.yml

```diff
@@ -19,6 +19,8 @@ jobs:
       with:
         java-version: adopt@1.11
     - run: npm install jsdom@12.2.0
+    - run: sbt stub-server/docker:publishLocal
+    - run: docker run -d -p 8080:8080 stubserver:1.0.0
```

Comment:
For some reason switching Scala versions kills background SBT processes, so this seems like the only way to get the StubServer running properly. Costs about a minute and a half in CI time though.